File: Java/Iniciante/1011 - Esfera.java
import java.util.Scanner;

import static java.lang.StrictMath.pow;

/*
 * Autor: Pierre Vieira
 * Data de submissão: 26/01/2020 14:14:37
 */
public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        double raio = teclado.nextFloat();
        double volume = 3.14159*pow(raio, 3)*(4.0/3);
        System.out.printf("VOLUME = %.3f\n", volume);
    }
}

File: C#/Iniciante/1011 - Esfera.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 13:25:41
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int raio = int.Parse(Console.ReadLine());
            double volume = (4.0 / 3) * 3.14159 * raio * raio * raio;
            Console.WriteLine($"VOLUME = {volume.ToString("F3", CultureInfo.InvariantCulture)}");
        }
    }
}
